HiP-DataStore.Model/Events/ReferenceEvents.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace PaderbornUniversity.SILab.Hip.DataStore.Model.Events
+{
+    public abstract class ReferenceEventBase : IEvent
+    {
+        /// <summary>
+        /// Collection where the referencing entity is in.
+        /// </summary>
+        public ResourceType SourceType { get; set; }
+
+        /// <summary>
+        /// ID of the referencing entity.
+        /// </summary>
+        public int SourceId { get; set; }
+
+        /// <summary>
+        /// Collection where the referenced entity is in.
+        /// </summary>
+        public ResourceType TargetType { get; set; }
+
+        /// <summary>
+        /// ID of the referenced entity.
+        /// </summary>
+        public int TargetId { get; set; }
+
+        public ReferenceEventBase()
+        {
```

Comment:
This pattern has come up several times here:

I don't like the fact that there is an empty contructor. I understand, why this yields the correct semantics, but I just think that the code looks like the result of a cut-and-paste-mistake or something. 

Mabye in some cases it is possible to use just the one constructor that actually contains code, and set the arguments to be optional.